c2corg_ui/templates/filters.html

```diff
@@ -0,0 +1,85 @@
+<%!
```

Comment:
Since this template is specific to waypoints, it should be moved to the ``c2corg_ui/templates/waypoint`` dir.